### ISSUE TITLE
The reference image should scale with the scene.

### ISSUE THREAD

**Issue by Ekrons** [umiyuki/Artstage-Issues#22]
https://youtu.be/MMsURDJ5OIU

When you model something by reference, and then you want to scale the scene, the aligned mesh and reference image get lost. There must be some kind of lock that locks and unlocks the movement of the image along with the scene.

**Comment by Ekrons** [comment]
I select an image and start modeling above it. I create edges along the contours of the image. Throughout the modeling, the image is behind the model.
Sometimes I need to enlarge the model along with the picture to add small details in model. But here the problem begins. When I enlarge the model, the image does not increase.
![GIF](https://user-images.githubusercontent.com/50044554/62949033-2872f780-bdee-11e9-9684-ce40a3cee5cc.gif)

**Comment by umiyuki** [comment]
There will be no need to do such work in VR. You can do it in blender. You can create with blender and import it into Artstage.

**Comment by Ekrons** [comment]
@umiyuki
It would be easier for me to do this in Artstage. I beg you to make this function. Is it really that hard?
Gravity Sketch already have this feature. There is a button in the corner with an image of an anchor. But Gravity Sketch less convenient for me than Artstage.

**Comment by Ekrons** [comment]
@umiyuki 
Here https://youtu.be/rPCNYj2tDRo?t=73

**Comment by umiyuki** [comment]
In the first place, your proposal is insufficient. Even if you try to add vertices on the reference image, if you move the cursor near the reference image, it will switch to the grab tool. You can't do this.

**Comment by Ekrons** [comment]
Here https://youtu.be/rPCNYj2tDRo?t=73

There is a lock in the layers that can prohibit influence on the image.

**Comment by Ekrons** [comment]
@umiyuki 
The link in the previous post is exactly what I want.

**Comment by umiyuki** [comment]
If you use a button to turn the anchor on or off, even if the cursor is pointed by chance, it will change to a pointer each time, which will get in the way.

**Comment by Ekrons** [comment]
@umiyuki 
And if you make a tick somewhere, for example, in layers, which will anchor and unanchor the image?
Or put this tick in the reference load menu. But then it will must work for all images in the scene. Which is worse than in layers.

**Comment by Ekrons** [comment]
@umiyuki 
In Artstage, in the layers there is a menu of objects where we can mirror or subdiv objects.
Why not make such a menu for images?

**Comment by umiyuki** [comment]
In any case, we still need to continue to consider.
At least it became clear that it was not easy to implement.

**Comment by Ekrons** [comment]
@umiyuki 
Yes...
- You have to add images as layers. To be able to block the impact on him.
- And you have to create a menu for the image, in which there will be a choice of how to scale it, with the scene or separately.
I hope you take up the implementation of this feature. As I said, I’ll pay.

**Comment by umiyuki** [comment]
This is not a deal. I just made your request clear only.
No payment is required and no implementation can be promised.

**Comment by Ekrons** [comment]
@umiyuki 
I also do not demand anything from you. I ask only to help facilitate the work, if you can. This feature is all a snag.
I don't like navigation in Blender. Honestly, after two years of work this already makes me sick. In VR, it’s much easier.
If you help, then all the work will be carried out at Artstage, which will facilitate the work and greatly accelerate the process.
Look how much time I spent at Artstage, it is very small. And all due to the fact that there is no this feature. When it appears, the hours spent at Artstage will grow progressively.
This will become the main program that I use.

**Comment by umiyuki** [comment]
No, you really demanded. You opened a duplicate issue #44 . You have to close that pointless issue. If such an act continues, you might becomes a banned.

**Comment by Ekrons** [comment]
@umiyuki 
Can you quote the message where I demand?
#44 was created not as a rquare thread, but as a discussion thread, where we could write any general questions.

I don’t understand for whom are you creating Artstage? If for yourself, then you do it well. If for users who are willing to work in this program, then not very good.
In the new update, convenient switching between tools has disappeared. Now I can’t use the program at all.
In general, you look at everything from your point of view, and you don't care about modelers.
I will no longer ask you for anything. Tired of it. You can not bother yourself with the ban, I'm leaving myself.

**Comment by umiyuki** [comment]
What is a general question thread? All issues need to be closed individually when resolved. Questions should be individual issues.
In that thread, You also demanded feature by writing "please do this feature."  This is not a question.
You repeat the act of screaming and leaving and eventually returning. There is no doubt that I am more tired of you than you. Good bye.

**Comment by umiyuki** [comment]
In addition, you asked to implement features at Twitter and Steam with a $ 20 or $ 8 reward, which is mostly an insult to the developer. It is ridiculous.

**Comment by MarkMironov** [comment]
I need this feature too.

**Comment by MarkMironov** [comment]
> In the first place, your proposal is insufficient. Even if you try to add vertices on the reference image, if you move the cursor near the reference image, it will switch to the grab tool. You can't do this.

No. Only the left hand acts on the reference image.

**Comment by MarkMironov** [comment]
The fact that the reference image cannot be moved together with the scene sometimes even brings inconvenience. I saved a scene with such an image, and when I open it again, the image was outside the playing area, I could not get it.

**Comment by MarkMironov** [comment]
@umiyuki, at how many dollars do you value this feature?

**Comment by MarkMironov** [comment]
@Ekrons, lets pool our money together and buy it.

**Comment by umiyuki** [comment]
@MarkMironov
Hello.
I was understanding that this feature is necessary. and continue to be positive about implementation.
The implementation of this feature is delayed only because we were thinking how best to implement it.
Your “make sure the reference images can only be grabbed by the left hand” is a good idea. However, if implemented as a reference image anyway, it is expected that the next request will be "I want you to be able to turn show and hide easily."
My current idea is to use it as a part of a sketch object. It's "sketch image". It is separate from the "reference image".
This makes it easy to turn show and hide and change transparency without interfering with modeling operations.
Maybe there are many people who need this feature, so I want to implement it early.
I think some additional features for example “snapping to a specific axis” and “projecting a sketch image onto a modeling object” may also be necessary, but they will take time to develop, so support may be added separately in the future.

**Comment by umiyuki** [comment]
@MarkMironov
Hello.
I was understanding that this feature is necessary. and continue to be positive about implementation.
The implementation of this feature is delayed only because we were thinking how best to implement it.
Your “make sure the reference images can only be grabbed by the left hand” is a good idea. However, if implemented as a reference image anyway, it is expected that the next request will be "I want you to be able to turn show and hide easily."
My current idea is to use it as a part of a sketch object. It's "sketch image". It is separate from the "reference image".
This makes it easy to turn show and hide and change transparency. Also these will not interfering with modeling operations.
there are seems many people who need this feature, so I want to implement it early.
I think some additional features for example “snapping to a specific axis” and “projecting a sketch image onto a modeling object” may also be necessary, but they will take time to develop, so support may be added separately in the future.

**Comment by MarkMironov** [comment]
@umiyuki Thanks!

> Your “make sure the reference images can only be grabbed by the left hand” is a good idea.
This is not an idea. This is what is already in the program. We can move the ref image with our left hand, and the right thumbstick is for resizing. The right trigger does not affect the ref image.

parsec3d wrote:
> Maybe a plane with texture on a new layer... so you can have a working layer in wire mode and the reference in the other layer with an image applied....

Thanks to you, parsec3d, I was able to display reference image texture on a plane. This is what I needed.
The only thing that is not very clear is how the unwrap inside Artstage works. But with the help of orher 3D modeling programs, I was able to import the unwraped plane and its texture.

**Comment by MarkMironov** [comment]
@umiyuki thank you!

**Comment by Ekrons** [comment]
@umiyuki thanks for the image feature.

@MarkMironov The program is still under development, and if I pay a lot of money for one feature, I will lose money and still be with a program that doesn't have many features I need.
Even a ZBrush license costs $ 900, but how rich this product is.
Therefore, I offered such small amounts for individual features. As a result, it would have turned out $ 300 from all the features I need.
It is unlikely that Umiyuki needs our money, especially since he already made this feature.

**Comment by umiyuki** [comment]
@Ekrons 
Ekrons, you complained and wrote in this thread. "I will no longer ask you for anything. Tired of it.", "I'm leaving."
But you are back and writing a new future request.
First of all, you should withdraw your remarks.
You did not even close ridiculous #44 that I asked you.
If your actions are not sincere, your remarks will not be respected.

**Comment by Ekrons** [comment]
@umiyuki primarily I didn't know that I had rights. I didn't find the "close issue" button.
If you noticed, I wrote a request to close the issue even in the issues that I created. Example #22. I don’t even know how to close it now.

I'm not a touchy and not vindictive person, unlike you, so my "leave" doesn't last forever.

And why is this branch ridiculous? Because you didn’t understand her?
On the contrary, it should be pinned.
Tell me, where should a person write a question that doesn't ask for a new feature, but just wonders how the existing feature works correctly? Or where should a person ask to focus the attention of a developer on a particular feature?
I believe that the topic of the problem should be a discussion of only its functions. And minor questions and discussions should have their own thread.
